src/main/java/org/opensearch/ad/ml/EntityColdStarter.java

```diff
@@ -268,15 +324,19 @@ private void trainModelFromDataSegments(List<double[][]> dataPoints, String mode
         }
 
         int rcfNumFeatures = dataPoints.get(0)[0].length;
-        RandomCutForest rcf = RandomCutForest
+        RandomCutForest.Builder rcfBuilder = RandomCutForest
             .builder()
             .dimensions(rcfNumFeatures)
             .sampleSize(rcfSampleSize)
             .numberOfTrees(numberOfTrees)
             .lambda(rcfTimeDecay)
             .outputAfter(numMinSamples)
-            .parallelExecutionEnabled(false)
-            .build();
+            .parallelExecutionEnabled(false);
+
+        if (rcfSeed != -1) {
```

Comment:
We are simply initializing it in AD so we have access to control what the seed is in our testing right? No behavioral change cause RCF would initialize a seed if not given one by AD?

Reply:
yeah, no behavior change will happen as by default we don't provide a seed.

Reply:
in 1.1 PR you set the seed too 2051L for the other tests that will use this setup. I assume it doesn't matter for the HCADModelPerfTests as we re-establish seed to be loop number. Should we add the same seed tho in the setup in the AbstractCosineDataTest class here?

Reply:
you meant to use loop number in the setup? The 2051L number will be used in other tests in EntityColdStarterTests. HCADModelPerfTests create a different EntityColdStarter and doesn't need the seed number 2051L.